.github/workflows/ci.yml

```diff
@@ -60,6 +59,7 @@ jobs:
     - name: Tag docker image
       run: docker tag docker.pkg.github.com/${OWNER}/${DOCKER_CACHE_REPO}/${DOCKER_TAG} ${DOCKER_TAG}
     - name: Check pre-commit
+      if: github.event_name != 'pull_request'
```

Comment:
I think we should add this to all the steps of the pre-commit job. That'll save time. Sadly github actions doesn't yet support skipping all the steps together, so it has to be in every step

Reply:
do you want @ziyiwu9494 to change this before I merge it?